widgets/social-media-buttons/social-media-buttons.php

```diff
@@ -194,25 +194,28 @@ function modify_instance( $instance ) {
 			}
 		}
 
-		if ( ! empty( $instance['design'] ) && ! isset( $instance['design']['icon_size_unit']) ) {
+		if ( ! empty( $instance['design'] ) && ! isset( $instance['design']['icon_size_unit'] ) ) {
+			$instance['design']['icon_size']      = $instance['design']['icon_size'] . 'em';
 			$instance['design']['icon_size_unit'] = 'em';
+			$instance['design']['rounding']       = $instance['design']['rounding'] . 'em';
 			$instance['design']['rounding_unit']  = 'em';
+			$instance['design']['padding']        = $instance['design']['padding'] . 'em';
 			$instance['design']['padding_unit']   = 'em';
 
 			// The margin value was previously changed based on the align setting.
-			$top = $right = $bottom = $left = $instance['design']['margin']. 'em';
+			$top = $right = $bottom = $left = $instance['design']['margin'] . 'em';
 			switch ( $instance['design']['align'] ) {
 				case 'left':
-					$left = '0';
+					$left = '0em';
```

Comment:
It isn't important but do we need `em` here?

Reply:
Yes. Without it, the value will be imported without a unit of measurement set.